packages/7guis-counter/src/routes/index.css

```diff
@@ -0,0 +1,36 @@
+body {
+  font-family: Gordita, Roboto, Oxygen, Ubuntu, Cantarell,
+    'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+main {
+  text-align: center;
+  padding: 1em;
+  margin: 0 auto;
+}
+
+h1 {
+  color: #335d92;
+  text-transform: uppercase;
+  font-size: 4rem;
+  font-weight: 100;
+  line-height: 1.1;
+  margin: 4rem auto;
+  max-width: 14rem;
+}
+
+p {
+  max-width: 14rem;
+  margin: 2rem auto;
+  line-height: 1.35;
+}
+
+@media (min-width: 480px) {
+  h1 {
+    max-width: none;
+  }
+
+  p {
+    max-width: none;
+  }
+}
```

Comment:
I see some files are missing a trailing newline, so would be helpful to add a Prettier config at the repo root at some point. Here's the one I made for my projects; it lists all the defaults for discoverability with customizations at the end in `overrides`: https://github.com/mangs/simple-release-notes-action/blob/main/config/prettier/prettierConfig.json

Reply:
Code formatting NPM scripts here: https://github.com/mangs/simple-release-notes-action/blob/main/package.json#L25-L29

Reply:
Code format validation NPM scripts here: https://github.com/mangs/simple-release-notes-action/blob/main/package.json#L36-L40